<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>981</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="OpenSerialButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>920</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="SendMsgEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>750</y>
     <width>761</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Send2CenterButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>920</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发送到中心</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>750</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>串口设置</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Sensorlabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;传感器&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="SensorcomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Checklabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;校检位&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CheckcomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Portlabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;波特率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="PortcomboBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>用户名</string>
   </property>
   <widget class="QPushButton" name="ChangenNameButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserNameEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>381</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>传感器数据</string>
   </property>
   <widget class="QTextEdit" name="SensorDataEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>351</width>
      <height>541</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>651</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>智能驾驶舱</string>
   </property>
   <widget class="QLabel" name="CarViewLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;这里有个汽车&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;这里有一辆车&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>  这里停放了一辆汽车</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>两公里内车辆数</string>
    </property>
    <widget class="QLCDNumber" name="CarCountlcd">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>速度盘</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="Press1Edit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Press2Edit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Press3Edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Press4Edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>油量</string>
    </property>
    <widget class="QProgressBar" name="OilBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>中心服务器</string>
   </property>
   <widget class="QLabel" name="NetConditionlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;网络状态：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearNetButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空网络消息</string>
    </property>
   </widget>
   <widget class="QLabel" name="IPlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NetPortlabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NetPortEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="NetStartButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>启动服务</string>
    </property>
   </widget>
   <widget class="QLabel" name="NetStatelabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="NetStopButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>断开服务</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="NetMsgEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>651</width>
     <height>171</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenFileButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>670</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveFileButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>670</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReadFileButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>670</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>读取数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>670</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询数据</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ContinueButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>继续</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PauseButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>660</y>
     <width>651</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QDial" name="Beauty1">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>390</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="Beauty2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>390</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox_9</zorder>
  <zorder>groupBox_8</zorder>
  <zorder>OpenSerialButton</zorder>
  <zorder>SendMsgEdit</zorder>
  <zorder>Send2CenterButton</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>NetMsgEdit</zorder>
  <zorder>OpenFileButton</zorder>
  <zorder>SaveFileButton</zorder>
  <zorder>ReadFileButton</zorder>
  <zorder>SearchButton</zorder>
  <zorder>Beauty1</zorder>
  <zorder>Beauty2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
